db/migrate/20171120000000_add_embed_columns_to_preview_cards.rb

```diff
@@ -0,0 +1,32 @@
+class AddEmbedColumnsToPreviewCards < ActiveRecord::Migration[5.1]
+  def change
+    status_ids = nil
+
+    reversible do |dir|
+      dir.up do
+        status_ids = Status.joins(:preview_cards)
+                         .where({ preview_cards: { type: :photo } })
+                         .reorder(nil)
+                         .group(:id)
+                         .pluck(:id)
+
+        PreviewCard.where(type: :photo).delete_all
+      end
+    end
+
+    add_column :preview_cards, :embed_url, :string, default: '', null: false
```

Comment:
I think that we should use `Mastodon::MigrationHelpers#add_column_with_default` that can add columns without locking.